resources/screen.css

```diff
@@ -418,7 +417,7 @@ h4 {
   padding: 0 2em;
 }
 
-#top-bar div.search {
+/* #top-bar div.search {
```

Comment:
Can this be removed now that there is style for ``.search``?

Reply:
We're gonna need css to remove the flash when it loads.

Reply:
But what does that have anything to do with this? You added different style for the new element, ``.search``.

Reply:
True.